### PR TITLE
new third parties and extra subdomains 

### PR DESCRIPTION
**Pull request by zeman**
Here's a few more services and extra sub domains we've found while digging through waterfall charts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -997,7 +997,7 @@ Lighthouse's bootup time audit attempts to attribute all toplevel main-thread ta
 
 ### The data for entity X seems wrong. How can it be corrected?
 
-Verify that the origins in `data/entities.json` are correct. Most issues will simply be the result of mislabelling of shared origins. If everything checks out, there is likely no further action and the data is valid. If you still believe there's errors, file an issue to discuss futher.
+Verify that the origins in `data/entities.json5` are correct. Most issues will simply be the result of mislabelling of shared origins. If everything checks out, there is likely no further action and the data is valid. If you still believe there's errors, file an issue to discuss futher.
```
Comment (patrickhulce) [inline README.md]: oh great catch thanks! could you make the change over in https://github.com/patrickhulce/third-party-web/blob/master/lib/markdown?

these will just get overwritten next time the readme gets generated

File: data/entities.json5 [hunk 2]
```diff
@@ -8508,8 +8524,9 @@
   },
   {
     "name": "Throtle",
+    "homepage": "https://throtle.io/",
     "categories": ["analytics"],
-    "domains": ["v12group.com"]
+    "domains": ["v12group.com", "thrtle.com"]
```
Comment (patrickhulce) [inline data/entities.json5]: this is intentional misspelling right? :)

File: data/entities.json5 [hunk 3]
```diff
@@ -824,12 +833,16 @@
...
-      "snap.licdn.com",
       "bizographics.com",
       "slideshare.com",
       "slidesharecdn.com"
     ]
   },
+  {
+    "name": "LinkedIn Ads",
+    "categories": ["ad"],
+    "domains": ["ads.linkedin.com", "snap.licdn.com", "www.linkedin.com"]
```
Comment (patrickhulce) [inline data/entities.json5]: can you add a comment (yay json5! :D) on the `www.linkedin.com` what sorts of paths are ads?

someday when it's path granularity it'd be nice to know where these came from :)

File: README.md (hunk 1 above)
Comment (zeman) [reply README.md]: Done.

File: data/entities.json5 (hunk 2 above)
Comment (zeman) [reply data/entities.json5]: Correct, that's the domain that use. I've also been cross checking against other projects like [Better. ](https://better.fyi/trackers/thrtle.com/)

File: data/entities.json5 (hunk 3 above)
Comment (zeman) [reply data/entities.json5]: Sweet, I'll add a comment. Here's the request we saw on wired.com...

Screen Shot 2019-04-30 at 2 13 34 PM

**Pull request merged**